test/lexers/vue-lexer.test.js

```diff
@@ -0,0 +1,31 @@
+import { assert } from 'chai'
+import VueLexer from '../../src/lexers/vue-lexer'
+
+describe('VueLexer', () => {
+  it('extracts keys from template & js', (done) => {
```

Comment:
I would prefer tests to be atomic and test a single thing as much as possible. This could be 2 tests.

Reply:
vue files consist of two parts. template & js. i18next can call from both parts. then test must cover  maximal case. I can add two tests for individual cases but the max case should be considered too.

what do you think?

Reply:
I'm fine with that.